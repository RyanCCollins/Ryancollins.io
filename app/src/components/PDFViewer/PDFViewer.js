import React, { PropTypes } from 'react';

const PDFViewer = ({
  url
}) => (
  <iframe
    src={`http://docs.google.com/gview?url=${url}&embedded=true`}
    style={{ width: '100vw', height: '100vh' }}
  />
);

PDFViewer.propTypes = {
  url: PropTypes.string.isRequired
};

export default PDFViewer;
