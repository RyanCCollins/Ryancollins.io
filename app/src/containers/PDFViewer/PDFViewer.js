import React from 'react';

const PDFViewer = ({
  url
}) => (
  <iframe
    src={`http://docs.google.com/gview?url=${url}&embedded=true`}
    style="width:600px; height:500px;"
    frameborder="0"
  />
);

export default PDFViewer;
