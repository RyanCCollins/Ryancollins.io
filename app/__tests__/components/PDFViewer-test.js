import React from 'react';
import renderer from 'react/lib/ReactTestRenderer';

import PDFViewer from '../../src/components/PDFViewer/PDFViewer';

// needed because of this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('<Divider />', () => {
  it('renders with default props', () => {
    const component = renderer.create(
      <PDFViewer
        url="https://s3.amazonaws.com/accredible-profile-uploads/udacity/resumes/1469994565898"
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
