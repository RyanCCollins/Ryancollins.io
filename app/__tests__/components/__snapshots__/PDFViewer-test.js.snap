exports[`<Divider /> renders with default props 1`] = `
<iframe
  src="http://docs.google.com/gview?url=https://s3.amazonaws.com/accredible-profile-uploads/udacity/resumes/1469994565898&embedded=true"
  style={
    Object {
      "height": "100vh",
      "width": "100vw"
    }
  } />
`;
